core-services/image-mirroring/knative/mapping_knative_v1_0_quay

```diff
@@ -12,6 +12,7 @@ registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-test-wathola
 registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-test-wathola-receiver quay.io/openshift-knative/wathola-receiver:v1.0
 registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-test-wathola-sender quay.io/openshift-knative/wathola-sender:v1.0
 registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-kafka-test-wathola-kafka-sender quay.io/openshift-knative/wathola-kafka-sender:v1.0
+registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-kafka-broker-test-kafka-consumer quay.io/openshift-knative/kafka-consumer:v1.0.0
```

Comment:
```suggestion
registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-kafka-broker-test-kafka-consumer quay.io/openshift-knative/kafka-consumer:v1.0
```

Reply:
We consume it in this way:
```
image_version=$(versions.major_minor "${KNATIVE_SERVING_VERSION}")
image_template="quay.io/openshift-knative/{{.Name}}:v${image_version}"
```
So it needs to be just major.minor version, not micro